packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -112,21 +118,18 @@ export function normalizeError(
     // We have to construct a new Fluid Error, copying safe properties over
     const { message, stack } = extractLogSafeErrorProperties(error, false /* sanitizeStack */);
     const fluidError: IFluidErrorBase = new SimpleFluidError({
-        errorType: "genericError", // Match Container/Driver generic error type
-        fluidErrorCode: "",
+        errorType: normalizeErrorType,
+        fluidErrorCode,
         message,
         stack,
     });
 
     fluidError.addTelemetryProperties({
         ...annotations.props,
         untrustedOrigin: 1, // This will let us filter to errors not originated by our own code
+        typeofError: typeof(error),
```

Comment:
This is going to be "object" 99% of the time, why not let it be blank?  I guess it's fine.